docs/guides/getting-started/concepts.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Concepts
+description: 'Get familiar with the terminology used in Latitude'
+---
+
+## Prompts
+
+Prompts are the core building block of Latitude. They are the questions or tasks that you want to ask your model. Prompts can be as simple as a single sentence or as complex as a multi-step dialogue. You can use prompts to generate text, classify text, or perform any other task that your model is capable of.
+
+In Latitude, prompts support parameters, chaining, shared snippets, and more advanced features like logic (if/else, loops) or version control.
+
+To find out more about prompts, check out the [Prompt manager](/guides/prompt-manager/) overview.
+
+## Logs
+
+Logs are the records of the interactions between your prompts and your model. They contain the input prompt, the model's response, and any other metadata that you choose to include. Logs are essential for evaluations, monitoring your model's performance, and debugging any issues that may arise.
```

Comment:
"They contain the input prompt, the model's response, and any other metadata that you choose to include" que dise

Reply:
It contains all info and metadata about the chain itself, yes. And, additionally, there's a "custom" field that the user can use with whatever text they want to, mainly thought for user ID or classification. Not sure if that's what you mean.

Reply:
is the sdk taking this into account? (passing custom values to logs)

Reply:
Not currently I think. There's no UI to filter them yet either, so it may not be that useful for now (?)

Reply:
yea that's what i meant by metadata

Reply:
@csansoon I think that field is now `text`. Should it be `json` column? Or have another called `metadata`? I would have one just for the identifier. 

Is very handy to be able to filter logs for an specific user in an easy way. If we want to do it in a json field it will be less performant and difficult to implement

Reply:
I like the idea of it being a `json` and just let the user filter by any property in that json.

I think we talked about it though. I don't remember why did we decide on using `text` for now though.